### ISSUE TITLE
Wont open an app.

### ISSUE THREAD

**Issue by Dikabro** [RishaLaser/Risha_Controller_V4.1#1]
When I trying to open this application Windows gives me an error.. :/

**Comment by Moushira** [comment]
@Dikabro hmm, Risha_Controller_V4.1/Paint/Paint/bin/Debug/Paint.exe doesn't open?  Which you are on win8?  Thanks :)

**Comment by Dikabro** [comment]
Nope, it doesn't open. I'm using x64 Windows 8 system. :)

**Comment by Moushira** [comment]
@Dikabro   yeah, it is probably the win8 thing.  Do you have access to another machine with win7 for testing..? Otherwise, you can use the Andriod app, if you have an android device.   Out of curiosity, did you build the machine already? :)

**Comment by Dikabro** [comment]
1. I have 4 PC's and they are all on windows 8..:D 
2. Don't have Android device. :(
3. I'm building my own laser engraving machine and writting some modifications to run it on Arduino Nano. So I just wanted to try your Paint application. ;)

**Comment by Dikabro** [comment]
Guys, why you decided to use Arduino Mega instead Uno? :)

**Comment by Moushira** [comment]
@Dikabro Mega and AVR were initial solutions, because we couldn't resolve the timers issue when we started :). This should change soon.  Will you share the Nano version when you finish it? :)  So what happens with win8..you get the protection warning?

**Comment by Dikabro** [comment]
As I'm working one I do things slow. But when I will finish it (I think after month) I will definitely post my work here on github. ;)

Now about what happens... When I start the program I get an error  "has stopped working" for about 1 second and then it just disappears. :)

**Comment by Moushira** [comment]
Great! Looking forward to the documentation...
Lets double check this error.  We are mostly Linux users, but lets see what happens on the Window$ land :)
